iceoryx_utils/test/moduletests/test_cxx_expected.cpp

```diff
@@ -175,6 +184,15 @@ TEST_F(expected_test, VoidCreateError)
     ASSERT_THAT(sut.get_error(), Eq(12.2f));
 }
 
+TEST_F(expected_test, VoidCreateErrorAndMove)
```

Comment:
I do not understand the Void in the test name. Clarify or change the name.

Reply:
I started reworking the existing test (see ed2db2)